deploy/docker-compose/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - QUEUE_SIZE=10
       - QUEUE_MAX_WORKERS=5
       - "CODEX_ADDRESS=http://gungnir:7000"
-      - "LOG_LEVEL=DEBUG"
+      - "LOG_LEVEL=error"
```

Comment:
We don't want debug logs?

Reply:
I think I got overwhelmed by the number of debug messages so I changed it to error while testing